tests/UseImportsResolver/UseImportsResolverTest.php

```diff
@@ -34,6 +36,6 @@ public function test(string $filePath, array $expectedClassUsages): void
 
     public static function provideData(): Iterator
     {
-        yield [__DIR__ . '/Fixture/FileUsingOtherClasses.php', [FirstUsedClass::class, SecondUsedClass::class]];
+        yield [__DIR__ . '/Fixture/FileUsingOtherClasses.php', [SomeFactory::class, FirstUsedClass::class, FourthUsedClass::class, SecondUsedClass::class]];
```

Comment:
Could you create a separate fixture file? Just so we have this clearly separated.

Reply:
Good idea, done!